Ryujinx.Cpu/MemoryManager.cs

```diff
@@ -684,7 +684,7 @@ public void SignalMemoryTracking(ulong va, ulong size, bool write)
             // Write tag includes read protection, since we don't have any read actions that aren't performed before write too.
             long tag = (write ? 3L : 1L) << PointerTagBit;
 
-            int pages = GetPagesCount(va, (uint)size, out va);
+            int pages = GetPagesCount(va, (uint)size, out ulong _);
```

Comment:
```suggestion
            int pages = GetPagesCount(va, (uint)size, out _);
```
No need to specify the type if the value will be discarded.